json-schema/input.yaml

```diff
@@ -94,6 +94,13 @@ definitions:
             type: number
           count:
             type: number
+      os:
```

Comment:
Please make sure this is sync'd with https://github.com/joyent/conch-json-schemas ... since the new field is gonna be required.